src/resources/Clusters/Cluster.ts

```diff
@@ -0,0 +1,27 @@
+import API from '../../APICore';
+import Resource from '../Resource';
+import {ClusterAgentModel, ClusterModel, ClusterStatusModel} from './ClusterInterfaces';
+
+export default class Cluster extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/clusters`;
+
+    list() {
+        return this.api.get<ClusterStatusModel[]>(Cluster.baseUrl);
+    }
+
+    listAgents(id: string) {
+        return this.api.get<ClusterAgentModel[]>(`${Cluster.baseUrl}/${id}/agents`);
+    }
+
+    live() {
+        return this.api.get<ClusterModel>(`${Cluster.baseUrl}/live`);
+    }
+
+    status(id: string) {
+        return this.api.get<ClusterStatusModel>(`${Cluster.baseUrl}/${id}/status`);
```

Comment:
This is the wrong type
![image](https://user-images.githubusercontent.com/35579930/67130095-5863c000-f1ce-11e9-8b8b-8178e302fac5.png)

Reply:
The type is correct (it was missing a property tho), but the name is different